marketplace_builder/pages/getting-started/notifications/notifications.liquid

```diff
@@ -0,0 +1,48 @@
+
+* `email_notifications`: contains all email notifications
+*  `sms_notifications`: contains all sms notifications
+*  `api_call_notifications`: contains all API call notifications
+
+## Configuration 
+
+Each configuration can accept different properties, but some of the main concepts are common:
+
+| Property            | Description                                                                                                                                                                                                        | Default | Example                                                                                                                                                                      |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `to`                | Depending on the alert type, it is either: {::nomarkdown}<ul><li>an email address of the recipient</li><li>mobile number of the recipient</li><li>endpoint for the API call</li></ul>{:/} Accepts liquid.          | n/a     | `test@email.com`                                                                                                                                                             |
+| `delay`             | Number of minutes by which the notification should be delayed.                                                                                                                                                     | 0       | 2                                                                                                                                                                            |
+| `enabled`           | Boolean which defines whether alert should be invoked or not. If false, it will be simply ignored.                                                                                                                 | true    | false                                                                                                                                                                        |
+| `trigger_condition` | Liquid condition to control whether notification should be sent or not. The only value that allows notification to be delivered is `true`. If you skip it, the notification will be sent (unless enabled is false) | true    | {% raw %}`{% if should_send == 'some_value'%}true{% endif %}`{% endraw %} - this code will trigger the notification only if value of `should_send` is equal to 'some_value'. |
+
+{% include alert/important content="`to` property is required in SMS and API Call notifications." %}
```

Comment:
{% include 'alert/important' content: "`to` property is required in SMS and API Call notifications." %}

Reply:
Thank you, fixed it!